Shopipy.ApiService/Program.cs

```diff
@@ -35,6 +39,9 @@
 
 builder.Services.AddAutoMapper(typeof(UserMappingProfile));
 
+builder.Services.AddScoped<IBusinessRepository, BusinessRepository>();
+builder.Services.AddScoped<BusinessService>();
```

Comment:
this logic belongs in business project, see [this](https://github.com/imbieras/shopipy/blob/kapi/authorization/Shopipy.UserManagement/DependencyInjection.cs)